<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SincroForm</class>
 <widget class="QDialog" name="SincroForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SincroForm</string>
  </property>
  <widget class="QLabel" name="label1job">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Job</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="field1Job">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label1Type">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="field1Type">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label1State">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>State</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="field1State">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label1TargetF">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target freq</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1RealF">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Real freq</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCD1TargetF">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCD1RealF">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar1DeviationP">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar1DeviationN">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label1Deviation">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Deviation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1WaitTime">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Wait time</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar1WaitTime">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>70</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="but1Start">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but1Stop">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>30</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but1DoTwo">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Do 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but1DoOne">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Do 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butON">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>520</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butOFF">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>520</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="field2Job">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar2WaitTime">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>170</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar2DeviationP">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label2Deviation">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Deviation</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar2DeviationN">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label2State">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>State</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCD2RealF">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label2RealF">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Real freq</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2job">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Job</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="field2Type">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label2WaitTime">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>170</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Wait time</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="field2State">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but2Stop">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>130</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCD2TargetF">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label2Type">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but2Start">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but2DoTwo">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>130</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Do 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but2DoOne">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Do 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2TargetF">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target freq</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>831</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label3job">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Job</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="field3Job">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar3WaitTime">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>270</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label3RealF">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>270</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Real freq</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but3DoTwo">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>230</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Do 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3WaitTime">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>270</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Wait time</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="field3State">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label3State">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>State</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3Deviation">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>270</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Deviation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3Type">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>230</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but3DoOne">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>230</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Do 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="field3Type">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="but3Start">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCD3TargetF">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar3DeviationP">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCD3RealF">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="Line" name="line2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>831</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label3TargetF">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target freq</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but3Stop">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>230</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar3DeviationN">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar4DeviationP">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label4TargetF">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target freq</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="field4Type">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCD4TargetF">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar4WaitTime">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>370</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="field4Job">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label4WaitTime">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>370</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Wait time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4RealF">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>370</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Real freq</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4Type">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>330</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4job">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Job</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="bar4DeviationN">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="but4DoOne">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>330</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Do 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="field4State">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label4Deviation">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Deviation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4State">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>State</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCD4RealF">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>370</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="but4Stop">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>330</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but4DoTwo">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>330</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Do 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but4Start">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="Line" name="line3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>831</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="butHeartUp">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>520</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>UP</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeart">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Heart rythm</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="LCDHeart">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="butHeartDown">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>520</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>DOWN</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
